posts/2019/06/python-library-method-usage.md

```diff
@@ -0,0 +1,217 @@
+<!--
+.. title: Method usage for popular numberical and scientific libraries
+.. slug: python-package-method-usage
+.. date: 2019-06-01
+.. author: Christopher Ostrouchov
+.. tags: Labs
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+Developers of open source software often have a difficult time
+understanding how others use their libraries. Having a better data of
+when and how methods are being used allows for better api design, more
+instructive tutorials, and understanding the adoption of new
+features.
+
+# Python Namespace Inspection
+
+A general tool
+[python-api-inspect](https://github.com/Quansight-Labs/python-api-inspect)
+was written to analyze any method/attribute call within a given set of
+namespaces within a repository. This work was heavily inspired by
+prior work inspecting method usage with [google big
+query](https://galeascience.wordpress.com/2016/08/10/top-10-pandas-numpy-and-scipy-functions-on-github/)
+for pandas, numpy, and scipy. The previously mentioned blog post used
+regular expressions to search for method usage. The primary issues
```

Comment:
issues --> you mention only 1 issue.

also, is that all? that's not too bad a limitation in practice I think. If there's a rule for `import numpy as np` then the results will be reasonably accurate.